script.js

```diff
@@ -0,0 +1,370 @@
+// Exercise javascript includes: 
+// - Template Strings
+// - Tag functions
+
+//****************************************************************
+
+
+
+
+//****************************************************************
+// -----Exercise Template Strings-----
+//****************************************************************
+
+const name = "Tom";
+const ordno= 12455;
+const total = 300.50;
+const msg = "Hello, "+name+" your \
+order("+ordno+") was :£"+total+".";
+
+console.log(msg); //Hello, Tom your order(12455) was :£300.5.
+
+
+const msg1 =
+  `Hello, ${name},  your
+order(${ordno}) was :£${total}.`;
+
+console.log(msg1);//Hello, Tom,  your
+                  //order(12455) was :£300.5.
+
+
+//****************************************************************
+
+//-----Exercise Tag function-----
+//****************************************************************
+
+function upper(strings,...values) {
```

Comment:
upper what? Can you name the function convertStringsToUpperCase or something more meaningful?